thirdeye/thirdeye-frontend/tests/utils/constants.js

```diff
@@ -98,6 +98,15 @@ export const rootCauseConst = {
   RCA_TOGGLE: '.rootcause-to-legacy-toggle'
 };
 
+/**
+ * General root-cause element selectors
+ */
+export const generalConst = {
+  MODAL: '.te-modal',
```

Comment:
Is there a reason why this can't be part of the rootCauseCOnst above if it's related to root cause?

Reply:
Oversight - these are general/global, not rootcause-specific.